src/vma/util/sys_vars.cpp

```diff
@@ -336,6 +336,41 @@ void mce_sys_var::read_env_variable_with_pid(char* mce_sys_name, size_t mce_sys_
 	}
 }
 
+bool mce_sys_var::check_cpuinfo_flag(const char* flag)
```

Comment:
@serglelv If you tx 512 Bytes (where BlueFlame is not involved) - is there illegal instruction or not?